tests/virtualization/universal/waitfor_guests.pm

```diff
@@ -102,15 +102,13 @@ sub run {
     add_guest_to_hosts $_, $virt_autotest::common::guests{$_}->{ip} foreach (keys %virt_autotest::common::guests);
     assert_script_run "cat /etc/hosts";
 
-    # Wait for guests to finish installation
-    # script_run("wait", timeout => 1200);
-    # script_retry("! ps x | grep -v 'grep' | grep 'virt-install'", retry => 120, delay => 10);
-    # Unfortunately this doesn't cover the second step of the installation, where ssh is available
-    my $sleep_delay = 1200;
-    $sleep_delay = 2400 if (is_xen_host);    # XEN has more guests
-    sleep($sleep_delay);    # XXX Get rid of this sleep!
-    wait_guest_online($_, 120) foreach (keys %virt_autotest::common::guests);
```

Comment:
This check should be removed as the new code will do it

Reply:
This code block is removed, since the PR was created and it is still removed.